db/schema.rb

```diff
@@ -66,6 +66,15 @@
 
   add_index "orangelight_call_numbers", ["sort"], name: "index_orangelight_call_numbers_on_sort", using: :btree
 
+  create_table "orangelight_name_titles", force: :cascade do |t|
+    t.text    "label"
+    t.integer "count"
+    t.text    "sort"
+    t.string  "dir"
+  end
+
+  add_index "orangelight_name_titles", ["sort"], name: "index_orangelight_name_titles_on_sort", using: :btree
+
```

Comment:
Is this table part of the barcode / cas login? If not, perhaps it should move to a different PR.